homeassistant/components/emoncms/manifest.json

```diff
@@ -2,5 +2,6 @@
   "domain": "emoncms",
   "name": "Emoncms",
   "documentation": "https://www.home-assistant.io/integrations/emoncms",
-  "codeowners": ["@borpin"]
+  "codeowners": ["@borpin"],
+  "version": "0.0.1"
```

Comment:
This versioning currently only applies to custom integrations, not core integrations. Core integrations are versioned by the Core itself.